server/src/main/java/org/opensearch/cluster/coordination/NoClusterManagerBlockService.java

```diff
@@ -76,6 +76,19 @@ public class NoClusterManagerBlockService {
         EnumSet.of(ClusterBlockLevel.METADATA_WRITE)
     );
 
+    /** @deprecated As of 2.2, because supporting inclusive language, replaced by {@link #NO_CLUSTER_MANAGER_BLOCK_ID} */
+    @Deprecated
+    public static final int NO_MASTER_BLOCK_ID = NO_CLUSTER_MANAGER_BLOCK_ID;
```

Comment:
Are we holding on to these because there are plugins or extensions that use this static variable outside of core?

Reply:
@nknize Thank you so much for your review! 👍
I didn't check whether the `public static` variables are used in plugins or not. My idea was to keep the compatibility in  minor versions, such as `2.x`, for all methods and variables that will be renamed.
This change is going to be backported into `2.x` branch, so I can't break compatibility for any `public` method or variable.
After doing a search in GitHub, looks like the variable `NO_MASTER_BLOCK_ID` is only used in this repository.

Reply:
I can't find usage of "NO_MASTER_BLOCK_" outside of the OpenSearch repo, so I think all of these are fair game for removal.

Reply:
Even changing the public variable name is a technical breaking change, after backporting this PR to `2.x` branch, is it fine to rename here directly instead of marking as deprecated first?

Reply:
@tlfeng you are taking the conservative approach and, IMHO, this is a good move. The inherited code base is sloppy wrt visibility modifiers and javadocs and this causes issues and confusion with extensions and external plugins. We're trying to fix that on OpenSearch to support our micro-repository project design. 

I presume these variables are "internal use only" thus safe to unilaterally refactor w/o compatibility support. However, since they have no markings I think we should treat them as if an extension is using them and follow the deprecation process.  I don't believe we have have this formally written out so I'll jot the process down here (you're largely already following this):

1. Update javadocs to communicate usage permissions (`@opensearch.internal`, `@opensearch.experimental`, or `@opensearch.api`) 
3. Add deprecation annotation
4. Update javadocs and deprecation messages
5. Commit deprecation and backport
6. Open a new PR that removes in main or refactors visibility modifier

Thoughts? /cc @andrross

Reply:
@nknize That is definitely the conservative approach and probably the right way to go. It's one extra PR with some trivial removals/changes so that's minimal extra work without having to worry about breaking our promise not to introduce breaking changes in a minor version.

Reply:
So then I think the only change to this PR would be to add `@opensearch.internal` to the new refactored variables.